<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XML_ui</class>
 <widget class="QMainWindow" name="XML_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1016</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XML_ui</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>252</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>252</width>
        <height>500</height>
       </size>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="Line" name="line_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>731</width>
        <height>441</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>223</red>
            <green>223</green>
            <blue>223</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>223</red>
            <green>223</green>
            <blue>223</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>223</red>
            <green>223</green>
            <blue>223</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="okButton_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="minimumSize">
         <size>
          <width>122</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>122</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>86</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="minimumSize">
           <size>
            <width>86</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>86</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Delete
Attribute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="minimumSize">
           <size>
            <width>162</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>162</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Add Child to Selected Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="minimumSize">
           <size>
            <width>162</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>162</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Delete the Selected Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="minimumSize">
           <size>
            <width>162</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>162</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Edit the Selected Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addAttButton">
          <property name="minimumSize">
           <size>
            <width>162</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>162</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Add
Attribute </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
